### ISSUE TITLE
Ability to load env even after build

### ISSUE THREAD

**Issue by batamar** [erxes/erxes-widgets#96]

**Comment by erdenezul** [comment]
@batamar it could be educational if you refer issue number in the commit message. So, the others can learn from the changes

**Comment by batamar** [comment]
@erdenezul Of course. I will mention this issue when I merge related pull request. I accidentally added committed label. And i removed it :))
